assessments/0003-krator.md

```diff
@@ -0,0 +1,181 @@
+        - **Contributing:** Include information on submitting issues and instructions with links on contributing to the code base and to the documentation.
+    - **API Documentation:** If you choose to bring this into the main documentation repo, the copy will need to be ported. Is the content available in markdown already? If not, it will need to be converted. 
+
+## Contributor documentation 
+
+| Criteria | 1 = (Is not present or requires significant work) | 2 | 3 = (is present, but needs work) | 4 | 5 = (is executed extremely well or no improvement required) |
+| ---                       | --- | --- | --- | --- | --- |
+| Communication methods documented  |  ✅  |     |     |     |     |
+| Beginner friendly issue backlog   |  ✅  |     |     |     |     |
+| “New contributor” getting started content  |  ✅  |     |     |     |     |
+| Project governance documentation |  ✅  |     |     |     |     |
+
+### Comments and recommendations
+
+Communication methods will need to be determined and added to [CONTRIBUTING.md](https://github.com/krator-rs/krator/blob/main/CONTRIBUTING.md).
+
+While there is an issue backlog, there aren't any issues with a `good first issue` label. As an invitation to new contributors, consider filing issues that are easy to remedy, labeling them accordingly, posting them on social media, and being available to help contributors through the process.  
+
+There is a brief mention of the PR review process in [CONTRIBUTING.md](https://github.com/krator-rs/krator/blob/main/CONTRIBUTING.md), however, giving more detail about the process, such as rounds of review, time frame, a link to [CODEOWNERS](https://github.com/krator-rs/krator/blob/main/CODEOWNERS), tests that have to pass, etc. help manage contributor expectations. This, along with linking to the Code of Conduct as you currently do, establishes a context for new contributors. 
+
+
+
+## Website
+
+| Criteria | 1 = (Is not present or requires significant work) | 2 | 3 = (is present, but needs work) | 4 | 5 = (is executed extremely well or no improvement required) |
+| ---                       | --- | --- | --- | --- | --- |
+| Branding and design  |  ✅  |     |     |     |     |
+| Case studies/social proof |  ✅  |     |     |    |     |
+| Maintenance planning |  ✅  |     |     |     |     |
+
+### Comments
+
+- **Branding and design**: Krator will need a logo if one doesn't exist yet. Along with the logo, well need a few colors, typically inspired by the logo, for backgrounds, links, and design elements such as borders.
+Sometimes Krator is capitalized, yet other times lowercase. I would recommend using one or the other consistently.
+
+- **Case studies/social proof**: A list of case studies or companies/organizations that use Krator would be an excellent addition to the home page. If adoption is still in its early phases, I'd recommend building out a place for logos that could be added in (or uncommented) as adoption rises.
```

Comment:
Because Krator is a new project, they likely won't have these available for a _long_ time. Instead, I'd consider:

```suggestion
- **Case studies/social proof**: As you develop Krator, keep an eye out for companies and projects using Krator, and start working with them early to do things like write blog posts about their experiences. As you gather user blog posts over time, submit a ticket to the CNCF Service Desk and we can turn these into a more marketing-friendly case studies section on your site.
```

Blog posts are a little more lightweight and easier to manage for a small project. This also makes it so that we don't build out a section that might take many months/years to become useful.